lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -125,6 +125,15 @@
     border: 1px solid $code-02;
   }
 
+  pre.code-block {
```

Comment:
Do we need the class here, or could we just blanket apply this to all `<pre>` elements within `.technical-documentation`?

Given any `pre` could be scrollable, it seems to make sense to tie the two together. Equally, if there are code blocks without a tabindex, the focus states won't apply to them anyway.

Reply:
Yeah, agreed. The class is not necessary.

Resolved in this force push: https://github.com/alphagov/tech-docs-gem/compare/67994d8b1e406b54e10587c57b228e359d1b5f22..0f5303b52c286d2aa3d5e51f65f9e651a5672fab